autoload/classpath.vim

```diff
@@ -0,0 +1,37 @@
+let s:entry = "\t<classpathentry kind=\"<kind>\" path=\"<path>\"/>"
+let s:entry_sourcepath = "\t<classpathentry kind=\"<kind>\" path=\"<path>\" sourcepath=\"<src>\"/>"
+
+" Creates a .classpath file and fills it with dependency entries
+function! classpath#generateClasspath() abort
+  let l:classes = extend(gradle#classPaths(), android#classPaths())
+  let classpath = ['<?xml version="1.0" encoding="UTF-8"?>', '<classpath>']
+  for jar in l:classes
+    call add(classpath, classpath#NewClasspathEntry('lib', jar))
+  endfor
+  call add(classpath, classpath#NewClasspathEntry('src', 'src/main/java'))
```

Comment:
Maybe we want to add all source paths? this would give you the source paths for all variants:

```
let l:srcs = extend(gradle#sourcePaths(), android#sourcePaths())
```

Reply:
I tried this at first, it didn't work with absolute paths.

Reply:
Would this work?

```
 for src in gradle#sourcePaths()
    let l:relativeSrc =  fnamemodify(src, ":s?" . gradle#findRoot() . "/??"
    call add(classpath, classpath#NewClasspathEntry('src', relativeSrc))
  endfor
```

Reply:
This works. Thanks.